apps/admin/_services/validation.service.ts

```diff
@@ -88,7 +93,7 @@ export class ValidationService extends BaseService {
 
     const innovationSupportedOnlyByUser = await em
       .createQueryBuilder(InnovationEntity, 'innovation')
-      .select(['innovation.id', 'innovation.name'])
+      .select(['innovation.id', 'innovation.name', 'supports.id', 'organisationUnit.name'])
```

Comment:
I see `supports.id` added to the select, but don't see it being used for the response. Should it be?

Reply:
It needs to be because of how the typeOrm works, innovation relates with support and support with orgUnit. Without any column from the support the mapper wouldn't map the field so the support would be empty. I'm not 100% sure it needs to be the id, think any column might work though but I always use the id in this case.